packages/core/src/__tests__/__e2e__/dropdown/Dropdown.cy.tsx

```diff
@@ -240,6 +241,36 @@ describe("Given a Dropdown", () => {
     cy.findByRole("button", { name: "end" }).should("be.focused");
   });
 
+  it("should not receive focus via mouse click if it is disabled", () => {
+    cy.mount(<Disabled />);
+    cy.findByRole("combobox").realClick();
+
+    cy.findByRole("combobox").should("be.disabled");
```

Comment:
This technically doesn't test focus only if it's disabled and the list doesn't open